packages/react-atlas-core/src/tableHeaderColumn/TableHeaderColumn.js

```diff
@@ -0,0 +1,43 @@
+import React, {PropTypes} from "react";
+import cx from 'classNames';
+import {TableHeaderColumn as BTableHeaderColumn} from "react-bootstrap-table";
+
+const TableHeaderColumn = ({className, children, ...props}) => {
+  return (
+    <BTableHeaderColumn {...props}>
```

Comment:
We should pass className to BTableHeaderColumn so users can override styles on BTableHeaderColumn. Also use cx so users can pass strings, arrays or objects on classNames.